### ISSUE TITLE
Is it normal for a stable coin to be worth $0.96?

### ISSUE THREAD

**Issue by Huper7777** [blockchain/My-Wallet-V3-Android#1307]
https://www.reddit.com/r/CryptoCurrency/comments/ulzirq/is_it_normal_for_a_stable_coin_to_be_worth_096/?utm_medium=android_app&utm_source=share

**Comment by chrisspradling1980** [comment]
Yes, as long as the stablecoin does not supercede past the country of
withdrawal currency amount. This is what prevents and maintains the
exemption of inflation with cryptocurrency.

**Comment by floreschristian** [comment]
Tnx

**Comment by ndevmetax** [comment]
> https://www.reddit.com/r/CryptoCurrency/comments/ulzirq/is_it_normal_for_a_stable_coin_to_be_worth_096/?utm_medium=android_app&utm_source=share

I had a similar issue and i sorted it out via live chat with customer support https://direct.lc.chat/14390013/
